lib/awful/placement.lua

```diff
@@ -603,6 +617,150 @@ local function rect_to_point(rect, corner_i, corner_j)
     }
 end
 
+-- Create a pair of rectangles used to set the relative areas.
+-- v=vertical, h=horizontal
+local function get_cross_sections(abs_geo, mode)
+print("\n\nLA!",mode)
```

Comment:
Uhm. Github GUI says that this is an outdated diff. However, I reviewed this by `git pull`ing your branch and looking at this locally and noticed the `print` there independently from Daniel. What's going on?!?